src/open_inwoner/accounts/models.py

```diff
@@ -616,7 +616,11 @@ def get_group_managed_categories(self):
     def save(
         self, force_insert=False, force_update=False, using=None, update_fields=None
     ):
-        if update_fields and "last_login" in update_fields:
+        if (
+            isinstance(update_fields, list)
+            and update_fields
+            and "last_login" in update_fields
+        ):
             update_fields.append("previous_login")
```

Comment:
The [docs say that](https://docs.djangoproject.com/en/4.2/ref/models/instances/#ref-models-update-fields):

> The update_fields argument can be any iterable containing strings. An empty update_fields iterable will skip the save. A value of None will perform an update on all fields.

Technically this means `update_fields` can be a whole bunch of things: `list`, `list`, generators, iterators, etc.

Given that Django doesn't care, I think we can safely convert to a list _first_, which will cover all types of iterable, so:

```suggestion
        if update_fields and "last_login" in update_fields:
            if not isinstance(update_fields, list):
                update_fields = list(update_fields)
                
            update_fields.append("previous_login")
```